a10_octavia/controller/housekeeping/house_keeping.py

```diff
@@ -27,6 +29,9 @@
 
 LOG = logging.getLogger(__name__)
 CONF = cfg.CONF
+mp_mgr = multiprocessing.Manager()
```

Comment:
Did we make sure the hk_ctx_map and hk_ctx_lock (in housekeeping process) is same instance as ctx_map/ctx_lock in a10_octavia_worker.py?

So, when housekeeping process lock the ctx_lock the controller worker threads will be locked?
(we can verify this by hard code some key in ctx_map inside housekeeper and see if we can see it in controller worker)

Reply:
yes, as we are assigning the value of hk_ctx_map and hk_ctx_lock to self.ctx_map and self.ctx_lock in the https://github.com/NehaKembalkarA10/a10-octavia/blob/STACK-3420/a10_octavia/controller/worker/controller_worker.py#L1506, the same value is used throughout the threads.